accord-core/src/test/java/accord/impl/mock/MockConfigurationService.java

```diff
@@ -18,27 +18,27 @@
 
 package accord.impl.mock;
 
+import accord.api.ConfigurationService;
 import accord.api.MessageSink;
 import accord.api.TestableConfigurationService;
 import accord.local.Node;
 import accord.primitives.Ranges;
 import accord.topology.Topology;
 import accord.utils.EpochFunction;
+import accord.utils.Invariants;
 import accord.utils.async.AsyncChains;
 import accord.utils.async.AsyncResult;
 import accord.utils.async.AsyncResults;
 
-import org.junit.jupiter.api.Assertions;
-
 import java.util.*;
 
 public class MockConfigurationService implements TestableConfigurationService
 {
     private final MessageSink messageSink;
     private final List<Topology> epochs = new ArrayList<>();
-    private final Map<Long, EpochReady> acks = new HashMap<>();
+    private final Map<Long, ConfigurationService.EpochReady> acks = new HashMap<>();
```

Comment:
static import class?

Reply:
Reverted all changes that are not related directly to the patch, including this one.